src/main/java/com/nvidia/spark/rapids/jni/RowConversion.java

```diff
@@ -121,6 +167,8 @@ public static Table convertFromRows(ColumnView vec, DType ... schema) {
   }
 
   private static native long[] convertToRows(long nativeHandle);
+  private static native long[] convertToRowsFixedWidthOptimized(long nativeHandle);
 
   private static native long[] convertFromRows(long nativeColumnView, int[] types, int[] scale);
+  private static native long[] convertFromRowsFixedWidthOptimized(long nativeColumnView, int[] types, int[] scale);
```

Comment:
I see two new JNI methods being declared here but no corresponding changes to RowConversionJni.cpp.

Reply:
Good catch. I believe I have this sorted, but I'm still having trouble testing myself.